packages/generator-single-spa/src/react/templates/tsconfig.json

```diff
@@ -1,8 +1,10 @@
 {
   "extends": "ts-config-single-spa",
   "compilerOptions": {
-    "jsx": "react-jsx"
+    "jsx": "react-jsx",
+    "declarationDir": "dist"
   },
-  "include": ["src/**/*", "node_modules/@types"],
+  "files": ["<%- mainFile %>"],
```

Comment:
The `"files"` configuration is necessary for typescript to know which file is the main file.